memote/suite/tests/test_thermodynamics.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Perform tests using eQuilibrator API on an instance of ``cobra.Model``."""
+
+from __future__ import absolute_import, division
+
+from sys import version_info
+
+import pytest
+
+if version_info[:2] < (3, 5):
+    pytest.skip("Thermodynamic tests require at least Python version 3.5.",
+                allow_module_level=True)
+
+import memote.support.thermodynamics as thermo # noqa
+import memote.support.basic as basic # noqa
+from memote.utils import (annotate, get_ids, truncate) # noqa
+
+
+@annotate(title="Thermodynamic Reversibility of Purely Metabolic Reactions",
+          format_type="percent")
+def test_find_incorrect_thermodynamic_reversibility(read_only_model):
+    u"""
+    Expect reversibility of metabolic reactions to agree with thermodynamics.
+
+    If a reaction is neither a transport reaction, a biomass reaction nor a
+    boundary reaction, it is counted as a purely metabolic reaction.
+    This test checks if the reversibility attribute of each reaction
+    in a list of cobrapy reactions agrees with a thermodynamics-based
+    calculation of reversibility. To determine reversibility we calculate
+    the reversibility index lngamma of each reaction
+    using the eQuilibrator API. The default cutoff for lngamma of 3
```

Comment:
Nice and clear explanation! It is a little 'mathy', though, so I suggest you take a look at the ['Notes' section 12](https://numpydoc.readthedocs.io/en/latest/format.html#sections) for inspiration with including tex. For example, it wasn't clear on first look that lngamma is the natural logarithm of gamma.

Reply:
Tex might actually look nice in the docs, but not so in the report which is where this string will end up in its raw form. Do you have any ideas how to still keep it readable?

I added an underscore for ln_gamma just like with the variable. That's the only part really requiring Tex right?

Reply:
Let me know if I can resolve this.

Reply:
Definitely keeping that in mind! I think that will be great to show how the score is calculated too!